### PR TITLE
Update create-a-keyword-dictionary.md

### PR DESCRIPTION
**Pull request by pratjain523**
@chrfox We have increased the limit of keyword dictionaries from 100KB to 1MB.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/compliance/create-a-keyword-dictionary.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ description: "Learn the basic steps to creating a keyword dictionary in the Offi
...
+Data loss prevention (DLP) can identify, monitor, and protect your sensitive items. Identifying sensitive items sometimes requires looking for keywords, particularly when identifying generic content (such as healthcare-related communication), or inappropriate or explicit language. Although you can create keyword lists in sensitive information types, keyword lists are limited in size and require modifying XML to create or edit them. Keyword dictionaries provide simpler management of keywords and at a much larger scale, supporting up to 1MB of terms (post compression) in the dictionary and support any language. The tenant limit is also 1MB after compression. 1MB of post compression limit means that all dictionaries combined across a tenant can have close to 1 million character.
   
 > [!NOTE]
 > Microsoft 365 Information Protection now  supports in preview double byte character set languages for:
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-keyword-dictionary.md:28]:
```suggestion
> Microsoft 365 Information Protection now supports in preview double-byte character set languages for:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/compliance/create-a-keyword-dictionary.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ description: "Learn the basic steps to creating a keyword dictionary in the Offi
...
+Data loss prevention (DLP) can identify, monitor, and protect your sensitive items. Identifying sensitive items sometimes requires looking for keywords, particularly when identifying generic content (such as healthcare-related communication), or inappropriate or explicit language. Although you can create keyword lists in sensitive information types, keyword lists are limited in size and require modifying XML to create or edit them. Keyword dictionaries provide simpler management of keywords and at a much larger scale, supporting up to 1MB of terms (post compression) in the dictionary and support any language. The tenant limit is also 1MB after compression. 1MB of post compression limit means that all dictionaries combined across a tenant can have close to 1 million character.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-keyword-dictionary.md:25]:
```suggestion
Data loss prevention (DLP) can identify, monitor, and protect your sensitive items. Identifying sensitive items sometimes requires looking for keywords, particularly when identifying generic content (such as healthcare-related communication), or inappropriate or explicit language. Although you can create keyword lists in sensitive information types, keyword lists are limited in size and require modifying XML to create or edit them. Keyword dictionaries provide simpler management of keywords and at a much larger scale, supporting up to 1 MB of terms (post-compression) in the dictionary and support any language. The tenant limit is also 1 MB after compression. 1 MB of post-compression limit means that all dictionaries combined across a tenant can have close to 1 million characters.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/create-a-keyword-dictionary.md:25]:
```suggestion
Data loss prevention (DLP) can identify, monitor, and protect your sensitive items. Identifying sensitive items sometimes requires looking for keywords, particularly when identifying generic content (such as healthcare-related communication), or inappropriate or explicit language. Although you can create keyword lists in sensitive information types, keyword lists are limited in size and require modifying XML to create or edit them. Keyword dictionaries provide simpler management of keywords and at a much larger scale, supporting up to  1 MB of terms (post-compression) in the dictionary and support any language. The tenant limit is also 1 MB after compression. 1 MB of post-compression limit means that all dictionaries combined across a tenant can have close to 1 million characters.
```

**Comment by yogkumgit** [comment]
@pratjain523

File: microsoft-365/compliance/create-a-keyword-dictionary.md (hunk 2 above)
Comment (pratjain523) [reply microsoft-365/compliance/create-a-keyword-dictionary.md:25]: I am good with the suggested changes.

**Pull request merged**